GitHubDashboard/Controllers/QueryController.cs

```diff
@@ -132,6 +132,39 @@ private async Task<IReadOnlyList<Issue>> GetIssuesAsync(string owner, string rep
             // Catch it in the calling function...
             var issues = await _gitHubClient.Issue.GetAllForRepository(owner, repository, issueRequest);
             issues = issues.Where(i => i.PullRequest == null).ToList();
+
+            // We now need to exclude the milestone
+            if (!String.IsNullOrEmpty(excludedMilestone) && !(excludedMilestone == "undefined"))
```

Comment:
Nit: Style is to use `string` over `String`.

Reply:
Also I would tend to use `excludedMilestone != "undefined"` over `!(excludedMilestone == "undefined")`

Reply:
fixed

---
In reply to: [247578233](https://github.com/Pilchie/GitHubDashboard/pull/13#discussion_r247578233) [](ancestors = 247578233)

Reply:
I was just following the style in the code. You can see line 123 which is the original. Would you like me to update the original? 

---
In reply to: [247578755](https://github.com/Pilchie/GitHubDashboard/pull/13#discussion_r247578755) [](ancestors = 247578755)

Reply:
Oh, yes, let's update that too.  Not sure how that got there.